### PR TITLE
{math}[intel/2018a] lavaan v0.6-2 w/ R 3.4.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2051.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6c129a3ef64131aae414e5369ba3751e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2408.golett.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a87cc87a4a58f82a277cb33229bbd296 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3140.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/48d71ce88e4477f3f029d9db1ab57681 for a full test report.

**Comment by boegel** [comment]
(re-triggering Travis checks...)

File: easybuild/easyconfigs/l/lavaan/lavaan-0.6-2-intel-2018a-R-3.4.4.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+homepage = 'https://cran.r-project.org/web/packages/%(name)s'
+description = """Fit a variety of latent variable models, including confirmatory factor analysis, structural
+ equation modeling and latent growth curve models."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = [
+    'https://cran.r-project.org/src/contrib/',
+    'https://cran.r-project.org/src/contrib/Archive/$(name)s/',
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/l/lavaan/lavaan-0.6-2-intel-2018a-R-3.4.4.eb]: this should be `%(name)s`

**Review by migueldiascosta — DISMISSED** [review]

File: easybuild/easyconfigs/l/lavaan/lavaan-0.6-2-intel-2018a-R-3.4.4.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/l/lavaan/lavaan-0.6-2-intel-2018a-R-3.4.4.eb]: fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/6856/commits/4c9391d74a57d43ff95a7f77c651547aa144f461

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node101.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/1476250676644937754d2ea619ea2e3c for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**